cmd/juju/application/config.go

```diff
@@ -463,33 +356,3 @@ func (c *configCommand) getConfig(client ApplicationAPI, ctx *cmd.Context) error
 	}
 	return errors.Trace(err)
 }
-
-// validateValues reads the values provided as args and validates that they are
-// valid UTF-8.
-func (c *configCommand) validateValues(ctx *cmd.Context) (map[string]string, error) {
```

Comment:
I moved `NewConfigCommandForTest` from `application/config.go` to `application/export_test.go`, since I think it makes a lot more sense there (this is the pattern followed by all other application commands).

Reply:
You're right tho, I did move `validateValues`, will move it back